### DISCUSSION TITLE
Login Session expired when creating a new Collection

### DISCUSSION THREAD

**Discussion by DaveSH12** [dani-garcia/vaultwarden#4959] [Q&A] [answered]
Hey all,
our Users are not allowed to create new Collections, but they have the Button to do this. When clicking the button they are automatically logged out with Message "Your Login session has expired".  Why this is happen and why there is no Message like "Sorry, you are not allowed to do this"?

Kind regards,
David

**Comment by stefan0xC** [comment]
What version? How do I reproduce that? I don't have that button as a User. And as a Manager it seems to work too (with the constraint of #4852).

**Reply by DaveSH12** [reply]
1.30.5

-
Web Installed
2024.1.2b
Database
SQLite: 3.44.0

My state is user and I have the Button "New" and "Collection".

**Reply by stefan0xC** [reply, marked as answer]
That version is outdated so this might have been fixed already. Please update to the newest release. Cf. https://github.com/dani-garcia/vaultwarden/issues/1180

**Reply by Gerardv514** [reply]
@stefan0xC is there a way to show in web vault and admin page, some sort of notification stating “there’s an update available” or something. Or what about an email notification to the admin or owner of the instance. 

There are so many similar threads like this that can be easily resolved if the instance is just updated.

**Reply by stefan0xC** [reply]
I think it already does in the `/admin/diagnostics` page and you can get also notified by GitHub about new releases. I mean we also inform about them in the [![Matrix Chat](https://img.shields.io/matrix/vaultwarden:matrix.org.svg?logo=matrix)](https://matrix.to/#/#vaultwarden:matrix.org). Keeping software up to date is also the users responsibility, especially when they include security fixes like the [last release](https://github.com/dani-garcia/vaultwarden/releases/tag/1.32.0).

**Reply by Gerardv514** [reply]
It is listed there but you will only see it if you proactively go into that. I’m talking about a way to proactively alert users that there’s an update, in hopes that would right in there faces alert to an update being available. Additionally, the Git alerts is only if they subscribe to that. It’s almost as if this should be forced to alert so that it’s known.

**Reply by DaveSH12** [reply]
Thanks @stefan0xC , Updating to the new version solved our problem
